excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/validation/ChallengeQuestionValidator.java

```diff
@@ -0,0 +1,208 @@
+package uk.gov.hmcts.ccd.definition.store.excel.validation;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.definition.store.domain.displaycontextparameter.DisplayContextParameterType;
+import uk.gov.hmcts.ccd.definition.store.domain.service.question.ChallengeQuestionTabService;
+import uk.gov.hmcts.ccd.definition.store.excel.endpoint.exception.InvalidImportException;
+import uk.gov.hmcts.ccd.definition.store.excel.parser.ParseContext;
+import uk.gov.hmcts.ccd.definition.store.excel.parser.SpreadsheetParsingException;
+import uk.gov.hmcts.ccd.definition.store.excel.parser.model.DefinitionDataItem;
+import uk.gov.hmcts.ccd.definition.store.excel.util.mapper.ColumnName;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.*;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+@Component
+public class ChallengeQuestionValidator {
+
+    private List<String> displayContextValues = Arrays.asList(new String[]{DisplayContextParameterType.DATETIMEDISPLAY.toString(), DisplayContextParameterType.DATETIMEENTRY.toString()});
```

Comment:
Is `DATETIMEDISPLAY` meant to be supported in the context of challenge questions?